packages/react/src/components/Modal/index.scss

```diff
@@ -0,0 +1,138 @@
+@use "../../utils/responsive" as *;
+
+.c__modal {
+  padding: 1.5rem;
+  background-color: var(--c--components--modal--background-color);
+  border-radius: var(--c--components--modal--border-radius);
+  border-color: var(--c--components--modal--border-color);
+  box-shadow: var(--c--components--modal--box-shadow);
+  color: var(--c--components--modal--color);
+  box-sizing: border-box;
+
+  &::backdrop {
+    // ::backdrop does not inherit from its element so CSS vars does not work.
+    // ( https://stackoverflow.com/a/77393321 ) So we need to use the color directly.
+    background: var(--c--components--modal--backdrop-color);
+    background: #0C1A2B99;
```

Comment:
If I understand well this is a fallback so I think this should above `background: var(--c--components--modal--backdrop-color);`

Reply:
It's a miss, in reality we are forced to use the hard coded value 😅 The `var` does not work at all.